test/screenshot/golden.json

```diff
@@ -679,6 +679,14 @@
       "desktop_windows_ie@11": "https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/09/05/20_29_19_047/spec/mdc-icon-button/classes/baseline.html.windows_ie_11.png"
     }
   },
+  "spec/mdc-icon-button/mixins/density.html": {
+    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/esgonzalez/2019/09/23/22_30_50_964/spec/mdc-icon-button/mixins/density.html?utm_source=golden_json",
```

Comment:
Can we render all density scales in the screenshot or at-least with 3 density scales?

Reply:
Got it!